distribution/kpkginstall/runtest.sh

```diff
@@ -466,6 +470,10 @@ else
       "${KVER}+debug"           # RHEL 8 style debug kernels
       "${KVER}.${ARCH}+debug"   # RHEL 8 style debug kernels
     )
+  elif grep -q "release 5" /etc/redhat-release; then
+    valid_kernel_versions=(
+      "${KVER_RHEL5}"
+    )
   else
     valid_kernel_versions=(
```

Comment:
Stupid question, but why doesn't this match the RHEL5 release since it has both `${KVER}` and `${KVER}.${ARCH}`? The second one looks like it should match the original NVR before `sed` :thinking:

Reply:
I thought the same thing but it was still expecting ${KVER}.${ARCH} ?  had to strip the aarch using sed, but could have been something else causing it to fail, I can retry removing this to see.
```
✅ Found kernel version string in cache on disk: 2.6.18-440.el5.x86_64
ℹ️ Acceptable kernel version strings: 2.6.18-440.el5 
ℹ️ Running kernel version string:     2.6.18-440.el5
```